src/TibiaDataUtils.go

```diff
@@ -0,0 +1,44 @@
+package main
+
+import (
+	"log"
+	"time"
+
+	"golang.org/x/text/unicode/norm"
+)
+
+// TibiadataDatetimeV3 func
+func TibiadataDatetimeV3(date string) string {
+	//TODO: Normalization needs to happen above this layer
+	date = norm.NFKC.String(date)
+
+	var (
+		returnDate time.Time
+		err        error
+	)
+
+	// If statement to determine if date string is filled or empty
+	if date == "" {
+		// The string that should be returned is the current timestamp in UTC
+		returnDate = time.Now().UTC()
```

Comment:
.UTC isn't needed here as it'll be done before the func returns

Reply:
nice catch! fixed :)